dlc_developer_config.toml

```diff
@@ -16,7 +16,7 @@ benchmark_mode = false
 
 [build]
 # Frameworks for which you want to disable both builds and tests
-skip_frameworks = []
+skip_frameworks = ["pytorch", "mxnet", "huggingface_tensorflow", "huggingface_pytorch"]
 # Set to false in order to remove datetime tag on PR builds
 datetime_tag = true
 # Note: Need to build the images at least once with datetime_tag = false
```

Comment:
can you run one revision with `safety_check_test = true` so we can check for python package vulnerabilities?